libraries/AP_Motors/AP_MotorsMatrix.h

```diff
@@ -86,6 +86,23 @@ class AP_MotorsMatrix : public AP_MotorsMulticopter {
     // add_motor using raw roll, pitch, throttle and yaw factors
     void                add_motor_raw(int8_t motor_num, float roll_fac, float pitch_fac, float yaw_fac, uint8_t testing_order, float throttle_factor = 1.0f);
 
+    // structure to statically store motor information.  Entries match
```

Comment:
It would be nice if we could move this MotorDefInt structure to be right before the add_motors() declaration in the .h file so that it is more understandable that this is only used for this method.

Reply:
public vs private issues with that

Reply:
Moved it.